internal/builder/_static/ecs_mappings.json

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
Should we record which version of ECS this is? And is it useful to know which version of ECS was applied to a given package? I am not sure how often ECS is updated.

Reply:
Currently, this set of dynamic templates are not related to a specific version of ECS. I would say it would not be needed to keep track of that.